apps/examples/src/hooks/usePerformance.ts

```diff
@@ -0,0 +1,23 @@
+import { TLUiEventSource, TLUiOverrides, debugFlags, measureCbDuration, useValue } from 'tldraw'
+
+export function usePerformance(): TLUiOverrides {
+	const measurePerformance = useValue(
+		'measurePerformance',
+		() => debugFlags.measurePerformance.get(),
+		[debugFlags]
+	)
+	if (!measurePerformance) return {}
+	return {
+		actions(_editor, actions) {
+			Object.keys(actions).forEach((key) => {
+				const action = actions[key]
+				const cb = action.onSelect
+				action.onSelect = (source: TLUiEventSource) => {
+					return measureCbDuration(`Action ${key}`, () => cb(source))
```

Comment:
in measureCbDuration, let's also make it a little prettier!

```
  console.debug('%c${name} took%c: ${performance.now() - now}ms', 'font-weight: bold; color: orange', 'font-weight: normal');
```

(different color here to make it easier to distinguish)